code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -14,19 +14,21 @@ emp_act
 		return FALSE
 	if(P.is_reflectable)
 		var/can_reflect = check_reflect(def_zone)
-		if(!can_reflect)
-			return (..(P , def_zone)) //Bad luck
-		else
-			if(can_reflect == 2) //If target is holding a toy sword
-				var/list/safe_list = list(/obj/item/projectile/beam/lasertag, /obj/item/projectile/beam/practice)
-				if(!is_type_in_list(P, safe_list)) //And it's not safe
-					return (..(P , def_zone)) //Bad luck
-		visible_message("<span class='danger'>The [P.name] gets reflected by [src]!</span>", \
+		if(can_reflect == 1) // proper reflection
+			visible_message("<span class='danger'>The [P.name] gets reflected by [src]!</span>", \
 						"<span class='userdanger'>The [P.name] gets reflected by [src]!</span>")
-
-		P.reflect_back(src)
-
-		return -1 // complete projectile permutation
+			P.reflect_back(src)
+			return -1 // complete projectile permutation
+
+		else if(can_reflect == 2) //If target is holding a toy sword
+			var/list/safe_list = list(/obj/item/projectile/beam/lasertag, /obj/item/projectile/beam/practice)
+			if(!is_type_in_list(P, safe_list)) //And it's not safe
+				return (..(P , def_zone)) //Bad luck
+			visible_message("<span class='danger'>The [P.name] gets reflected by [src]!</span>", \
+						"<span class='userdanger'>The [P.name] gets reflected by [src]!</span>")		
+			P.reflect_back(src)
+
+			return -1 // complete projectile permutation
```

Comment:
I know you didn't add/change this but, I have some have some concerns about the possible early return in here on line 26. Before the toy sword reflect PR, if something failed to reflect, no matter what, it would continue on through the rest of the proc. There was no early return like this. But now, it can terminate here, skipping some important stuff below about shields and whatnot. So, I think this might need to be something like:

```dm
	else if(can_reflect == 2) //If target is holding a toy sword
		var/list/safe_list = list(/obj/item/projectile/beam/lasertag, /obj/item/projectile/beam/practice)
		if(is_type_in_list(P, safe_list)) //And it's not safe
			visible_message("<span class='danger'>The [P.name] gets reflected by [src]!</span>", \
						"<span class='userdanger'>The [P.name] gets reflected by [src]!</span>")		
			P.reflect_back(src)
			return -1 // complete projectile permutation
```

Reply:
yeah could fix that, didn't think it would be much of an issue unless you considered adding more toys with reflect